src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,51 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface CardProps {
```

Comment:
make it simple, just use `Props` everywhere
```suggestion
interface Props {
```